src/components/LogRow/ResmokeRow/index.tsx

```diff
@@ -26,7 +27,7 @@ const ResmokeRow = forwardRef<any, BaseRowProps>((rowProps, ref) => {
       <CollapsedRow ref={ref} {...listRowProps} numCollapsed={line.length} />
     );
   }
-  const lineContent = getLine(line);
+  const lineContent = escapeHtml(getLine(line) || "");
```

Comment:
I wonder if we should do this when processing the resmoke lines in `LogContext/state.ts`. What do you think? 🤔

Reply:
I moved it to baseRow instead. All the characters are technical real characters so we need to be able to search them and replacing them in `LogContext/state.ts` will not allow us to do that anymore.